packages/react/src/Suspense.spec.tsx

```diff
@@ -5,17 +5,17 @@ import { createElement } from 'react'
 import { beforeEach, describe, expect, it } from 'vitest'
 import { Suspense, withSuspense } from '.'
 
-const resetBeforeEach = () => beforeEach(Suspend.reset)
+const resetBeforeEach = () => beforeEach(() => Suspend.reset())
```

Comment:
I'm wondering what you're getting by modifying it, is it unnecessary anonymous function declarations and calls?

Reply:
https://typescript-eslint.io/rules/unbound-method/

I don't think it's really problem but, I just want to avoid their eslint error if there is chance to make error by it.

<img width="639" alt="image" src="https://github.com/suspensive/react/assets/61593290/ba2eaa4c-b044-419a-8f47-6d6bacc05935">

Reply:
Thanks for the explanation, I didn't realize that was an eslint setting! Thanks for the confirmation!